windowing_library.py

```diff
@@ -59,13 +55,13 @@ def start_time_on_smaller_windows(windowed_data, threshold):
   return start_time[0][3][0]
 
 def find_precise_start_time(lift_windows, rows, window_size, stride, variance_threshold):
-	front = lift_windows[0]
-	np_front = np.array(front)
-	windowed_data = view_as_windows(np_front, (rows, window_size), step=stride)
-	start_time = start_time_on_smaller_windows(windowed_data, variance_threshold)
-	return start_time
+  front = lift_windows[0]
```

Comment:
follow pep8 style: https://www.python.org/dev/peps/pep-0008/#indentation
each indent stage in python should be 4 spaces